src/components/account-menu/template.njk

```diff
@@ -9,7 +11,10 @@
       <a href="#" class="hmrc-account-menu__link">Back</a>
     </li>
     <li>
-      <a href="#" class="hmrc-account-menu__link">Messages<span class="hmrc-notification-badge">2</span>
+      <a href="#" class="hmrc-account-menu__link">Messages
```

Comment:
By introducing a line break you're introducing a space into the content - I think that's a good thing, just want to make sure you're aware of this change.

Reply:
yep, that was the idea 😄 it's much tidier than having a space followed by the conditional all on one line